appserver/payara-appserver-modules/microprofile/healthcheck/src/main/java/fish/payara/microprofile/healthcheck/HealthCheckService.java

```diff
@@ -104,6 +109,9 @@ public void postConstruct() {
             events = Globals.getDefaultBaseServiceLocator().getService(Events.class);
         }
         events.register(this);
+        this.backwardCompatibilityEnabled = ConfigProvider.getConfig()
+                .getOptionalValue("mp.health.enable-backward-compatibility", Boolean.class)
```

Comment:
This flag is specifically for the Payara not part of MicroProfile Health spec.

Reply:
We might want to use another name since the `mp.<spec>.*` usually is used for constants in the spec. In FT they also don't use the canonical names for configuration values but simple names like `MP_Fault_Tolerance_Metrics_Enabled`. Can't say for the other MP parts but it would be nice if there was some consistency in the naming scheme.